doc/stages/writers.ept_addon.rst

```diff
@@ -0,0 +1,66 @@
+.. _writers.ept_addon:
+
+writers.ept_addon
+=================
+
+The **EPT addon writer** supports writing additional dimensions to `Entwine Point Tile`_ datasets.  Thus the EPT addon writer is unique among most other PDAL writers due to its limitation that it may only be used along with an `EPT reader`_ and modifies and existing dataset rather than creating a brand new one.
+
+The addon dimensions created by this writer may be stored independently from the corresponding EPT dataset, therefore write-access to the EPT resource itself is not required to create and use addon dimensions.
+
+.. embed::
+
+Example
+--------------------------------------------------------------------------------
+
+This example downloads the Autzen dataset (10M points) and runs the `SMRF filter`_, which populates the ``Classification`` dimension with ground values, and writes the resulting attribute to an EPT addon dataset on the local filesystem.
+
+.. code-block:: json
+
+   [{
+      "type": "readers.ept",
+      "filename": "http://na.entwine.io/autzen/ept.json",
+      "threads": 12
+   }, {
+      "type": "filters.assign",
+      "assignment": "Classification[:]=0"
+   }, {
+      "type": "filters.smrf"
+   }, {
+      "type": "writers.ept_addon",
+      "addons": { "~/entwine/addons/autzen/smrf": "Classification" },
+      "threads": 12
+   }]
+
+And here is a follow-up example of reading this dataset with the `EPT reader`_ - with the created addon overwriting the ``Classification`` value - and then writing the output to a single file with the `LAS writer`_.
+
+.. code-block:: json
+
+   [{
+      "type": "readers.ept",
+      "filename": "http://na.entwine.io/autzen/ept.json",
+      "addons": { "Classification": "~/entwine/addons/autzen/smrf" },
```

Comment:
It wouldn't be a horrible idea to show an example or at least mention how multiple addons can be read/written by the EPT drivers.

Reply:
Added colorinterp RGB example in c57730d.